From d178317f866d51c1c0873c171dc66aaff599155e Mon Sep 17 00:00:00 2001
From: Yabin Cui <yabinc@google.com>
Date: Sun, 5 Jun 2022 16:09:37 +0300
Subject: [PATCH 2/3] Fix error for compiler update.

frameworks/native/services/inputflinger/reader/mapper/SensorInputMapper.cpp
:306:23: error: declaration shadows a local variable [-Werror,-Wshadow]
    for (const auto& [sensorType, sensor] : mSensors) {
                      ^

Bug: 193903230
Test: build with the next compiler
Change-Id: I8087db738cfbef82e8264bbbc22515361bd11bf1
(cherry picked from commit 823144f333f8976eb98f28d8b802d1a82e6442fb)
---
 services/inputflinger/reader/mapper/SensorInputMapper.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/inputflinger/reader/mapper/SensorInputMapper.cpp b/services/inputflinger/reader/mapper/SensorInputMapper.cpp
index 7ac2dec895..a507632d0e 100644
--- a/services/inputflinger/reader/mapper/SensorInputMapper.cpp
+++ b/services/inputflinger/reader/mapper/SensorInputMapper.cpp
@@ -303,7 +303,7 @@ bool SensorInputMapper::setSensorEnabled(InputDeviceSensorType sensorType, bool
      * the device
      */
     mDeviceEnabled = false;
-    for (const auto& [sensorType, sensor] : mSensors) {
+    for (const auto& [_, sensor] : mSensors) {
         // If any sensor is on we will turn on the device.
         if (sensor.enabled) {
             mDeviceEnabled = true;
-- 
2.36.1

